### ISSUE TITLE
Can not get tablet to recognize pen at all. 

### ISSUE THREAD

**Issue by milowerx** [georgejecook/BostoTabletDriverMac#5]
The unit turns on works and I can see the screen fine but the pen does not connect to the tablet. the light is on on the pen as well so i know it's working.  I am using the 19" tablet but honestly don't know which version. Any help would be greatly appreciated!

**Comment by georgejecook** [comment]
Ok.. I'll need to know which version of the tablet you have.

Also - in the tablet options, in the first tab (called "general") what does it say?
Does it say connected?
Which version of os x do you have?

**Comment by milowerx** [comment]
It says Not connected and there doesnt seem to be a way to connect it. Also, they even sent me more than one pen but neither seems to work. Bosto sent me these units to review for my site http://www.animationinsider.com which is why i don't know much about which tablet they sent. they sent me a 19" one and a 14" one.

**Comment by milowerx** [comment]
I'm fairly certain I have an MA tablet.

**Comment by milowerx** [comment]
Oka then... so I discovered that there is only one USB port on the device that the USB cord can be plugged into (USB IN) so I now have the pen being recognized but it's very far offset and is not really useable. Is there no way to calibrate it on the Mac? On the PC there is a calibration tool in the Control panel for the Tablet PCs. But I do not see that sort of equivalent on the Mac.

**Comment by georgejecook** [comment]
Yes,

Click on the bostotablet icon, then select the "offset" tab..

You can edit it there.. it's fiddly ( I need to improve it)..

But.. 
which version of mac os x are you using?
Which tablet are you using?
Is the offset consistent across the screen, or does it vary?

**Comment by milowerx** [comment]
I am using 10.8.2 I'm fairly certain I have the MA version as the MB version according the Bosto site has a smooth side and the MA has a rigged side. The Offset is consistent is really only relegated to the top 1/4 of the screen. I can't even really maneuver around on the other screen.and the offset tool does not really work. it does not move my cursor downward towards the pen's location if that's how it's supposed to work. Might it have something to do with the fact that I already have a Cintiq? Do I need to uninstall the driver for the Cintiq? I know on the PC is was this way.

**Comment by milowerx** [comment]
Ok I have figured out that part of the problem is that the Kingtee wants to be known as the primary monitor. Once I switched it to that the cursor was at least on the same screen. I can not get the offset to do much though. Is there a tutorial on how it works? It's not very intuitive. At least not to me! :-)

**Comment by georgejecook** [comment]
Ok, thanks.. 

the drivers on mac don't interrupt each other.

Can you please open the bosto window and clarify that it actually says : "connected".

I'd really appreciate a screen grab on youtube..

Over the next couple of weeks I'll add more features to help people debug.

Please bare in mind this is an open source driver - I made this so my artist could use her 19mb on her mac.. I'm not officially supporting bosto at all.. but if you give me clear info; I'll do my best to fix things for you if I get time.

It might be that if you have time next week, I will make a special build available to you to get me debug info I need for the ma.

It sounds like my driver is not totally ma friendly.

**Comment by georgejecook** [comment]
You are correct - I should provide some more guidance on how to use the tablet.

I'll update with a big help button next week :)

And yes - you should select which of your monitors is the bosto and only use that with the pen.. I toyed with the idea of using the pen to span to monitors; but it was just a bit ugly.

**Comment by milowerx** [comment]
Oh I totally understand you're unofficial and I think what you're doing is pretty awesome so sorry if I made it seem like you are supposed to help me. No worries at all man. I can give you a video on youtube sure but I'm not sure what you'd see except an offset cursor. And yes I can confirm that I do in fact see the pen is connected.

**Comment by milowerx** [comment]
Hah, well BOSTO should provide more guidance on how to use the tablet. The material is pretty sparse! They should also be paying you to do this driver. Most artists I work with are Mac users. Making a Windows only tablet is a bit short sighted.

**Comment by georgejecook** [comment]
That's why I made the mac driver.

I'm actually rooting for them. They're starting out; they will get their shit together I'm sure.. Bosto wasn't built in a day ;)

At the end of the day - I think having a cheap cintiq alternative is a godsend for the community - so I'm happy to help - please don't take my comments as being irritated : I'm just managing expectations..

The fact you are reviewing it makes me want to solve the issues too, so you can give it a fair spin. I'd like to see this become more prominent.. more sales will definitely help the company focus and improve their retail channels.

So... if your offeset is now consistent across the screen, you can easily fix that by going to the "offset" tab..

unfortunately you have to tap, tap tap tap, keep tapping to change the offset. I forgot to make it work with just holding the mouse down.. but I will add that later in the week.

Pressure  is editable in the drawingoptions, as is the smoothing. You have a test pad their to practice to get it just right too..

Let me know if that helps you out.

And contact me about all other issues. I'm just a bit busy these last 2 weeks but I full intend to return and improve the driver for all the mac community to enjoy - I'm a mac guy too, as are ALL designers I know... I think Bosto being russia/china based has made them blind to the appeal their tablet would get in the mac market.. Let's try and help them see that :)

There are several milestones in the github project if you want to see my plans; I'm open to more suggestions too.

**Comment by milowerx** [comment]
I'm rooting for them as well. As I said I run a fairly popular website called Animation Insider which interviews artists in the animation, vfx and gaming industries each day but I am also in the animation industry myself and have a Cintiq both at work and home. Anyway, Cintiqs are beautiful pieces of machinery but expensive as Hell.
I totally agree they'll get their shit together at some point but it definitely aint there yet. I have a Hackintosh which lets be boot into either mac or PC and the PC version of the drivers ins better but still fraught with confusion. I feel like it's implementation of the button software is unusable at best for the average artist. As for the Offset tool on your driver I tried clicking the offset button but it did not seem to move it much at all. I'm not really sure how to make it work though. Is the idea I am supposed to use the mouse to do it or the stylus? If I use the mouse I can't really see much movement at all, if I use the stylus it moves like one pixel a click but I still don't see it move much at all. maybe it's just me.

**Comment by georgejecook** [comment]
That''s the limitation.. one click == one pixel move..

so you click click click, it moves 3 pixels, click click 2 more..

it's a bit painful to setup.. but using the 4 arrows, with enough clicks you can get it where you want it.. in the next version I'll add an autotargetting thingy.. so you just click a point on the screen and it will automatically calibrate.

for now though, you have to press the left, right, up or down button again and again and again until your cursor is comfortable..

**Comment by Niall01** [comment]
Hi, i'm not exactly very technical minded here, but i had the same problem and as a short term solution i have found if your offset is still off by a lot then a suggestion would be what i did, now this is just a short term solution for me you don't have to take this into consideration because im sure it's probably more hassle in the long run than is needed, i'm just using it this way until George has had enough time to develop his driver more,

The way i have got my offset as close as i possibly can is i have stopped using the VGA cable to my mac and instead have started using HDMI to mini display port that or DVI to mini display, this way when you are in your display settings in your system preferences it gives you an option for "Overscan" and i have found it works much better with mirrored displays as well with your bosto as the primary monitor, Now if you turn overscan off it will change your display size by a good bit, but in result it will keep your cursor under the pen and it only strays by a few millimetres but nothing to much to complain about in my case. you will lose some drawing estate to work with but it's an ok solution if you have enough time with the tablet to wait for some updates on the driver.  This might be an invalid solution in some cases but i though I would throw it in there for whatever little help it may serve.

**Comment by Niall01** [comment]
Also for what i just suggested you need to have the driver installed but the driver menu closed for some reason, i think it's because of the offset problems with the X and Y axis on the driver i'm not sure but that is how i got mine to work to the best of it's capabilities in these early stages of development

**Comment by milowerx** [comment]
Hmmm, I was not able to get the HDMI cable to work at all on my Mac but it's most likely because as i said I have a Hackintosh which is really a PC not a Mac that's been hacked to run OSX. I am using the DVI cable though and will have to look for the overscan button. I'll let you know what happens!

**Comment by Niall01** [comment]
No problem hope it helps, it crops down your screen by a good wee bit, but it's better than have your cursor going every odd direction.

**Comment by pramo69** [comment]
i have pretty much same issue , i am using mac mini and my pad is the 16HD Bosto software was installed but does not say pen or pad is active? Can you direct me in the right direction?

**Comment by pramo69** [comment]
i should have said connected

**Comment by tjallenn123** [comment]
hello, i am on a pc with a Bosto drawing tablet but the pen isn't responding. is there any way to fix it?
